draft-ietf-quic-transport.md

```diff
@@ -1000,6 +1000,12 @@ to more quickly identify when a connection becomes unusable.
 Packets that are matched to an existing connection, but for which the endpoint
 cannot remove packet protection, are discarded.
 
+Invalid packets without packet protection, such as Initial, Retry, or Version
+Negotiation, SHOULD be discarded without altering connection state.  An endpoint
```

Comment:
I tend to agree with @marten-seemann. I wonder if we might rather say MAY here (or "can"), because people might wonder what the anticipated behavior is, without the definition of what "invalid" means (does it mean a packet with a broken encoding? or does it mean a frame containing a semantically invalid frame; i.e. ACK containing a PN that was never used); and my understanding is that we do not want to define "invalid."